README.md

```diff
@@ -36,22 +25,28 @@ learning frameworks.
-  > **NOTE**: Limited support for TensorFlow models. The models created using Sequential or Keras Functional API are only supported.
-- Common interface for compression methods.
-- GPU-accelerated layers for faster compressed model fine-tuning.
-- Distributed training support.
-- Configuration file examples for each supported compression algorithm.
-- Git patches for prominent third-party repositories ([huggingface-transformers](https://github.com/huggingface/transformers)) demonstrating the process of integrating NNCF into custom training pipelines
-- Exporting PyTorch compressed models to ONNX\* checkpoints and TensorFlow compressed models to SavedModel or Frozen Graph format, ready to use with [OpenVINO&trade; toolkit](https://github.com/openvinotoolkit/).
-- Support for [Accuracy-Aware model training](./docs/Usage.md#accuracy-aware-model-training) pipelines via the [Adaptive Compression Level Training](./docs/accuracy_aware_model_training/AdaptiveCompressionLevelTraining.md) and [Early Exit Training](./docs/accuracy_aware_model_training/EarlyExitTraining.md).
+All of these algorithms can be executed through [Accuracy-Aware model training](./docs/Usage.md#accuracy-aware-model-training) pipelines: [Adaptive Compression Level Training](./docs/accuracy_aware_model_training/AdaptiveCompressionLevelTraining.md) and [Early Exit Training](./docs/accuracy_aware_model_training/EarlyExitTraining.md). 
+They allow to achieve the maximum compression level or to reduce the fine-tuning time, while staying within the range of the user-defined maximum accuracy degradation.
+
+## Post-Training Optimization Algorithms
+
+|Compression algorithm|PyTorch|TensorFlow|   ONNX    |OpenVINO|
+|:--------------------| :---: | :---: |:---------:| :---: |
+| Quantization        | Supported | Supported | Supported | Supported |
+
+## Usage of Compression-Aware Training Algorithms
 
-## Usage
-The NNCF is organized as a regular Python package that can be imported in your target training pipeline script.
-The basic workflow is loading a JSON configuration script containing NNCF-specific parameters determining the compression to be applied to your model, and then passing your model along with the configuration script to the `create_compressed_model` function.
-This function returns a model with additional modifications necessary to enable algorithm-specific compression during fine-tuning and handle to the object allowing you to control the compression during the training process:
+The basic workflow steps:
+1) Load an optimization configuration.
+2) Pass the original model along with the optimization configuration to the `create_compressed_model` function. `create_compressed_model` function returns a model with additional modifications necessary to enable algorithm-specific compression during fine-tuning and handle to the object allowing you to control the compression during the training process.
```

Comment:
Rephrase it, please. For example:
```suggestion
2) Pass the original model along with the optimization configuration to the `create_compressed_model` function. This function returns a model with additional modifications necessary to enable algorithm-specific compression during fine-tuning. In addition, this model contains handles that allow you to control compression in the learning process.
```

Reply:
The initial phrase is correct. It means that there are two objects that are returned: model and controller, which is used in fine-tuning.
But it could be rephrased as it confused you

Reply:
The sentence is too long to clearly understand the pipeline, IMHO.

Reply:
This construction might be helpful: 

```suggestion
2) Pass the original model along with the optimization configuration to the `create_compressed_model` function. `create_compressed_model` function returns (1) a model with additional modifications necessary to enable algorithm-specific compression during fine-tuning and (2) a handle to the object allowing you to control the compression during the training process.
```

Reply:
done